hyperbook/book/spiele/penguins/xml.md

```diff
@@ -246,8 +253,16 @@ Der Status der oben in dem data memento objekt enthalten ist, ist beispielsweise
 </state>
 ```
 
-Es enthält die Zugnummer in ``turn``, das anfangende Team in ``startTeam``, eine Liste an Feldern in ``board`` und die Anzahl Fishe jedes Spielers in ``fishes``.
-In `board` entsprechen die Positionen der Felder auf dem Spielfeld ihrer Positionen in der Liste => `x` Koordinate und der Position ihrer Liste im Board => `y` Koordinate. Die Zahl innerhalb der `field` objekte steht für die Anzahl Fishe auf dem Feld.
+Es enthält die Zugnummer in ``turn``, das anfangende Team in ``startTeam``, eine Liste an Feldern in ``board`` und die Anzahl Fishe beider Spielers in ``fishes``, zuerst die von ONE und danach die von TWO.
+
+In `board` entsprechen die Positionen der Felder auf dem Spielfeld in [odd-r Koordinaten](https://www.redblobgames.com/grids/hexagons/#neighbors-offset) ihrer Positionen in der XML Struktur. Von oben bei null anfangend werden die `<list>` Objekte in `<board>` durchgezählt. Die Nummer der List in dem das Field Objekt ist, ist seine `y` Kooridnate.  Auf die gleiche Weise entspricht die `x` Koordinate eines Felds seiner Nummer innerhalb des `<list>` Objekts.  
+Der Inhalt der `field` Objekte steht für die Anzahl Fishe auf dem Feld, oder für einen Pinguin eines Spielers. In ersterem Fall steht eine Zahl in dem XML-Objekt und in letzterem Fall der Name des Spielers der den Pinguin gehört.
+
+`x`: `<field>` index in `<list>`
+
+`y`: `<list>` index in `<board>`
+
+Beachtet, dass der XML-Status [odd-r Koordinaten](https://www.redblobgames.com/grids/hexagons/#neighbors-offset) nutzt, und dass die Koordinaten in Zügen als [doubled Koordinaten](https://www.redblobgames.com/grids/hexagons/#neighbors-doubled) interpretiert werden.
```

Comment:
> Beachtet, dass der XML-Status [odd-r Koordinaten](https://www.redblobgames.com/grids/hexagons/#neighbors-offset) nutzt

Inwiefern? Da werden doch keine Koordinaten mitgegeben?

Reply:
Die Koordinaten, die ich da meine, sind die, die in den Zeilen weiter oben beschreiben werden, aka die Position des <field> tags in der <boad> Struktur.